packages/flutter_tools/lib/src/android/android_device.dart

```diff
@@ -436,6 +436,20 @@ class AndroidDevice extends Device {
     runCheckedSync(cmd);
     return true;
   }
+
+  @override
+  bool get supportsScreenshot => true;
+
+  @override
+  Future<bool> takeScreenshot(File outputFile) {
+    const String remotePath = '/data/local/tmp/screen.png';
```

Comment:
Yup, I borrowed this path from some existing bits of our adb code; I updated the file name we put in there to `flutter_screenshot.png`.